.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "vsicons.presets.angular": false
```

Comment:
You can add `.vscode` to [.gitignore](https://github.com/mozilla/addons-frontend/blob/master/.gitignore) to prevent it from happening.